solidjs-tailwind/src/components/UserProfile/UserProfile.spec.jsx

```diff
@@ -0,0 +1,30 @@
+import { Router } from '@solidjs/router';
+import { render } from 'solid-testing-library';
+import { beforeEach, describe, expect, it } from 'vitest';
+import UserProfile from './UserProfile.jsx';
+import { userProfileProps } from './data';
+
+describe('User profile card', () => {
+  let wrapper;
+  beforeEach(async () => {
+    wrapper = await render(() => (
+      <Router>
+        <UserProfile {...userProfileProps} />
+      </Router>
+    ));
+  });
+
+  it('should mount', () => {
+    expect(wrapper).toBeTruthy();
+  });
+
+  it('should show the user display name', async () => {
+    const fullName = await wrapper.getByText(userProfileProps.name);
+    expect(fullName).toBeVisible();
+  });
+
+  it('should have a link for user profile picture', async () => {
+    const avatar = await wrapper.getByAltText('Avatar');
```

Comment:
does everyone have profile picture? I don't think so

Reply:
GitHub gives them a placeholder if they don't upload a profile picture. That placeholder is returned as the profile picture when we query their data.